Telerik.JustMock/Core/MocksRepository.cs

```diff
@@ -803,11 +807,23 @@ internal void AssertAction(string message, Action memberAction, Args args = null
 		{
 			using (MockingContext.BeginFailureAggregation(message))
 			{
-				var callPattern = ConvertActionToCallPattern(memberAction);
+				var callPattern = ConvertActionToCallPattern(memberAction, true);
```

Comment:
The public asserts API using this method are not fully covered with unit tests and allowing dispatch to method mocks can be dangerous.

Reply:
@mihail-vladov, fixed with [fddf417](https://github.com/telerik/JustMockLite/pull/39/commits/fddf417f417b749148a45ea863e9b37874674e50)